### PR TITLE
Update tokenlist.json - ADD Evmoon and Light Tokens

### PR DESCRIPTION
**Pull request by CosmosElon**
Projects from CoXen / Codxn community. (XEN Crypto on Evmos)

# Project Description

Projects from CoXen / Codxn community. (XEN Crypto on Evmos) 
https://t.me/XENCryptoEvmos
https://twitter.com/xenvmos

## Checklist

- [X] Contract is verified
- [X] Token is launched on Evmos mainnet / testnet
- [X] Token is an ERC20

### CODE REVIEW

**Comment by LPX55** [comment]
@CosmosElon Contracts need to be verified on escan. This is to prevent rug-pullable tokens from being listed. Let me know if you need assistance.

**Comment by CosmosElon** [comment]
Ok. Both tokens are verified now.

File: src/tokenlist.json [hunk 1]
```diff
@@ -480,6 +480,22 @@
...
+      "address": "0x524400756F3D288d90C04eaA0d130cB9c4ADC412",
+      "logoURI": "https://dweb.link/ipfs/bafybeia3zgw7y7eephx7h2b4hkhffjnvgfjjhtsfpbkumaf5mfj4y7ncbi/photo_2023-12-20_12-27-55.jpg"
+    },
+    {
+      "name": "Light",
+      "symbol": "LIGHT",
+      "chainId": 9001,
+      "decimals": 6,
+      "address": "0xe7520B4b89F105c53D4DBA596ebaF1Bd6A44D544",
+      "logoURI": "hhttps://upload.wikimedia.org/wikipedia/commons/1/1a/White_pog.svg"
```
Comment (LPX55) [inline src/tokenlist.json:498]: Fix logo URL

**Comment by CosmosElon** [comment]
done

**Pull request closed without merging**